tests/conftest.py

```
@@ -11876,6 +11878,127 @@ def finalizer():
+    region = cld_mgr.aws_client.region
+    created = {}
+
+    def _cleanup():
+        # If the test failed while creds were disabled, re-enable them so
+        # NooBaa can reach the target bucket during backingstore deletion
+        if "access_key_id" in created:
+            try:
+                aws_obj.update_access_key_status(
+                    created["username"], created["access_key_id"], active=True
+                )
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)